build.zig

```diff
@@ -83,20 +83,32 @@ pub fn build(b: *std.Build) void {
         .filter = if (b.args) |args| args[0] else null, // filter tests like so: zig build test -- "<FILTER>"
     });
 
+    const hard_fork_tests = b.addTest(.{
```

Comment:
you shouldnt need to modify the build file -- should remove these changes -- after adding the file path to `lib.zig`, running `zig build test` should run the tests you added